### PR TITLE
Add Kudo Spark 3.0.0-1.1.0

### PR DESCRIPTION
**Pull request by alexeygorobets**

### CODE REVIEW

File: addons/spark/1.x/spark-3.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+kind: Addon
+metadata:
+  name: spark
+  labels:
+    kubeaddons.mesosphere.io/name: spark
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "3.0.0-1"
+    appversion.kubeaddons.mesosphere.io/spark: "3.0.0"
+    catalog.kubeaddons.mesosphere.io/kudo-params: "https://raw.githubusercontent.com/kudobuilder/operators/master/repository/spark/operator/params.yaml"
```
Comment (akirillov) [inline addons/spark/1.x/spark-3.yaml:12]: let's use the release commit sha instead of master

**Review by akirillov — DISMISSED** [review]
let's update the params file location to commit sha instead of master. Apart from that LGTM 👍

**Review by akirillov — APPROVED** [review]
Thanks, @alexeygorobets. Confirming that the addon installs and works without issues on the latest RCs of Konvoy and Kommander. Let's merge it

**Comment by akirillov** [comment]
The `integration-test` fails on Cassandra:
```
2021-02-04T00:09:41.455916772Z === CONT  kuttl
2021-02-04T00:09:41.455934733Z     harness.go:382: run tests finished
2021-02-04T00:09:41.455940197Z     harness.go:486: cleaning up
2021-02-04T00:09:41.455944501Z     harness.go:541: removing temp folder: ""
2021-02-04T00:09:41.456140631Z --- FAIL: kuttl (513.62s)
2021-02-04T00:09:41.456148667Z     --- FAIL: kuttl/harness (0.00s)
2021-02-04T00:09:41.456153205Z         --- FAIL: kuttl/harness/cassandra-install-0-x (510.86s)
2021-02-04T00:09:41.456160314Z FAIL
```

The `Enterprise Addons` CI job fails on Spark test, however it looks like an issue in the test itself. Anyway, I'd suggest resolving/revamping Spark-related tests first prior to merging. Link to the Spark issue in TC: https://teamcity.mesosphere.io/buildConfiguration/kubeaddons_EnterpriseAddons/3055955?buildTab=log&focusLine=699&linesState=271

File: addons/spark/1.x/spark-3.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: gcp
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
```
Comment (kaiwalyajoshi) [inline addons/spark/1.x/spark-3.yaml:25]: Do we need this?

**Review by kaiwalyajoshi — APPROVED** [review]

File: addons/spark/1.x/spark-3.yaml (hunk 2 above)
Comment (alexeygorobets) [reply addons/spark/1.x/spark-3.yaml:25]: AFAIK we need it to be able to deploy on-prem and docker environments

**Comment by alexeygorobets** [comment]
I've launched the Spark test locally and they are passed.
There is probably a CI environment issue.
ticket for visibility: https://jira.d2iq.com/browse/D2IQ-74079

**Pull request merged**